tc/sgx/trusted_worker_manager/enclave/enclave_data.cpp

```diff
@@ -19,18 +19,26 @@
 #include <stdlib.h>
 #include <string>
 #include <vector>
+#include <string.h>
+#include<mbusafecrt.h>
 
 #include "crypto.h"
 #include "error.h"
 #include "avalon_sgx_error.h"
 #include "tcf_error.h"
 #include "zero.h"
+#include "enclave_utils.h"
 
 #include "jsonvalue.h"
 #include "parson.h"
 
 #include "enclave_data.h"
 
+// Below constants are the max limit for serialized version(base64 encoded)
```

Comment:
Good coding practice is to keep these constants in header file.

Reply:
Ok, will move it to header file.

Reply:
It depends. If these constants/pre-processor symbols has be shared across multiple source files then keeping them in header file is suggested. But if these are to be used only locally then keeping them in source files is the better approach.

Reply:
This constant is only used by enclave_data.cpp to calculate string length safely.